packages/formation-react/src/components/AcceptTermsPrompt/AcceptTermsPrompt.stories.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import AcceptTermsPrompt from './AcceptTermsPrompt';
+
+export default {
+  title: 'Library/AcceptTermsPrompt',
+  component: AcceptTermsPrompt,
+  argTypes: {
+    title: { control: 'text' },
+    headerContent: { control: 'text' },
+    footerContent: { control: 'text' },
+    yesContent: { control: 'text' },
+    termsContent: { control: 'text' },
```

Comment:
I don't think you'll need these if you add them to `AcceptTermsPrompt.propTypes`. This is used as an override for the controls in Storybook.

Reply:
yep you're right ... according to the docs [here](https://storybook.js.org/docs/react/writing-stories/args#component-args) it says `argTypes` can be used to globally override props so ill go ahead and remove `argTypes` for this component